src/cf-tables/src/cf-tables.less

```diff
@@ -81,20 +97,24 @@
 }
 
 .respond-to-min( @bp-sm-min, {
-    .table__striped {
+    .o-table__striped {
         .striped-table();
     }
-});
+} );
 
 .respond-to-max( @bp-xs-max, {
-    .table__striped > tbody > tr:nth-child(even) {
+    .o-table {
+        width: 100%;
+    }
+
+    .o-table__striped > tbody > tr:nth-child(even) {
```

Comment:
Could we change this to `.o-table__striped tr:nth-child(even)`  to be a little bit less specific and faster?